presto-product-tests/src/main/java/com/facebook/presto/tests/cassandra/CassandraUtils.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests.cassandra;
+
+import com.google.common.base.Stopwatch;
+import com.teradata.tempto.query.QueryResult;
+
+import java.util.concurrent.TimeUnit;
+
+import static com.teradata.tempto.query.QueryExecutor.query;
+import static java.lang.String.format;
+import static org.testng.Assert.fail;
+
+public class CassandraUtils
+{
+    private CassandraUtils() {}
+
+    public static void waitUntilTableCreation(String tableName, long timeoutMillis)
```

Comment:
This method is fine, but i think it would be better to introduce `assertContainsEventually` method in product test for such `eventual` assertion purposes. Same as we do in integration tests: `QueryAssertions#assertContainsEventually`.